modules/get-started/pages/licensing.adoc

```diff
@@ -0,0 +1,31 @@
+= Enterprise Licensing 
+:description: Learn how to purchase and apply an Enterprise Edition license.
+
+Anyone can use the certified and community connectors, but you need an Enterprise Edition license to use Enterprise connectors. 
+
+== Purchase an Enterprise Edition license
+
+You can purchase and download a license from the http://redpanda.com/try-enterprise[Redpanda website^].
+
+== Apply a license key to Redpanda Connect
+
+Once you have downloaded a license key, you can apply it in the following ways:
+
+. Save the license key to any local directory and run the following command:
```

Comment:
```suggestion
. Save the license key to any local directory and run the following command:
```

This line isn't needed for the `--redpanda-license` flag.

Reply:
Updated